server/api/region_test.go

```diff
@@ -344,7 +348,7 @@ func (s *testRegionSuite) TestSplitRegions(c *C) {
 	mustRegionHeartbeat(c, s.svr, r1)
 	mustPutStore(c, s.svr, 13, metapb.StoreState_Up, []*metapb.StoreLabel{})
 	newRegionID := uint64(11)
-	body := fmt.Sprintf(`{"retry_limit":%v, "split_keys": ["%s","%s","%s"]}`, 0,
+	body := fmt.Sprintf(`{"retry_limit":%v, "split_keys": ["%s","%s","%s"]}`, 3,
```

Comment:
Do we have a test case to test the default value of the `retry_limit`?

Reply:
No, I test it manually.